dts/bindings/misc/zephyr,morse-code.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2024 Freedom Veiculos Eletricos
+# Copyright (c) 2024 OS Systems
+# SPDX-License-Identifier: Apache-2.0
+
+description: Morse Code
+
+compatible: "zephyr,morse-code"
+
+include:
+  - name: base.yaml
+
+properties:
+  speed:
```

Comment:
This is a value that is direct performed by the driver to send the data. It can be changed dynamically too. So, in this case I prefer keep it on devicetree. Usually it is better have the timing details in the HW spec since can change from one scenario to another.

Reply:
Right that's fair :) Didn't really think about the dynamic aspect of it.